eventapi.py

```diff
@@ -0,0 +1,60 @@
+import json
+from datetime import datetime
+from flask import request
+from flask_restful import Resource, abort
+
+from event import Event, DAYS
+
+def read_args(request):
+    return json.loads(request.data.decode('utf-8'))
+
+def read_time(time_str):
+    return datetime.strptime(time_str, '%H:%M:%S').time()
+
+def validate_days(days):
```

Comment:
simplified, these could be placed into a validations file that could be imported, that way the implementation remains the same, but the logic is abstracted from the Event Application